src/hooks/useAuthFlow.ts

```diff
@@ -1,9 +1,11 @@
 import { useCallback, useContext } from 'react'
+import { createFetchComponent } from '@well-known-components/fetch-component'
```

Comment:
Please do not use this createFetchComponent. This will bring you some troubles. You should use fetcher in shared folder